### PR TITLE
Same button mapping as EdgeTX default.

### PR DESCRIPTION
**Pull request by L1cardo**
1. button mapping is now the same as EdgeTX default.
~~2. channel 5, 6, 7, and 8 were mapped both to analog axis and joystick buttons before, and there would be some issues on some sims, so now they are only mapped to analog axis.~~

### CODE REVIEW

File: Sketchbook/CRSF_Joystick/CRSF_Joystick.ino [hunk 1]
```diff
@@ -30,7 +31,7 @@ void packetChannels()
 {
     // Manually expanding instead of looping so I can change params as needed
     
-    // X - Channel 1 - A
+    // A - Channel 1 - X
```
Comment (mikeneiderhauser) [inline Sketchbook/CRSF_Joystick/CRSF_Joystick.ino:34]: Do you have a link to the EdgeTX default mapping?

Comment (L1cardo) [reply Sketchbook/CRSF_Joystick/CRSF_Joystick.ino:34]: https://github.com/EdgeTX/edgetx/blob/a04a56c0e7a657925b42e8f3e36035a357e5ed5e/radio/src/targets/common/arm/stm32/usbd_hid_joystick.c#L102

https://github.com/EdgeTX/edgetx/blob/a04a56c0e7a657925b42e8f3e36035a357e5ed5e/radio/src/targets/common/arm/stm32/usbd_hid_joystick.c#L127

**Comment by froqstar** [comment]
> led blinks when connection is down(same as ELRS).

I quite liked the LED being off when there is no connection, enables you to have the dongle plugged in all the time but the TX only when needed without having the bright LEDs on all the time. 
ExpressLRS allows you to disable the LED (by unassigning the LED pin).

**Comment by L1cardo** [comment]
> > led blinks when connection is down(same as ELRS).
> 
> I quite liked the LED being off when there is no connection, enables you to have the dongle plugged in all the time but the TX only when needed without having the bright LEDs on all the time. ExpressLRS allows you to disable the LED (by unassigning the LED pin).

LED blinks so that you know your CRSFJoystick is not broken, or at least are working.
If you really want LED off, we can make it blink for 60 seconds and then off.

**Comment by L1cardo** [comment]
@froqstar LED now blinks only for 60 sec when connection is down.https://github.com/mikeneiderhauser/CRSFJoystick/pull/12/files#diff-1010fe37319d62af747e6ed783eb39fc403e49ee7b8412660918f4b687b15081R142

**Comment by froqstar** [comment]
Thanks a ton ❤️

**Comment by mikeneiderhauser** [comment]
The led blink routine as implemented does not do what you think it does.  In order for me to consider this PR, the following changes need to be made.

- Update the led blink routine to be non blocking (FSM with millis() would be fine and quick to implement), have a configurable timeout (at compile time is fine), and be a self contained feature (#define enabled) at compile time.  You don't need to use the pico led to blink if the onboard elrs led is already blinking. I know your working on a different hardware rev and can see why you want this, but needs to be configurable and enable.
- Restore button count to 8

**Comment by L1cardo** [comment]
Closed since no plan to merge.

**Pull request closed without merging**